scripts/core/editor3/helpers/highlights.js

```diff
@@ -486,6 +486,33 @@ export function getRangeAndTextForStyle(editorState, style) {
     };
 }
 
+export function handleBeforeInputHighlights(onChange, chars, editorState) {
```

Comment:
This function is missing `@ngdoc`, I think it's nice to put it when we export a function

Reply:
I only put these for more complex functions. Do you use any tooling which depends on `@ngdoc`?

Reply:
not really, but in case someone does